setup.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Setup file for easy installation."""
+import glob
+
+from rpmlint import __version__
+from setuptools import setup
+
+setup(
+    name='rpmlint',
+    description='RPM file QA correctness validator',
+    long_description='Command-line tool for RPM files QA validation',
```

Comment:
Rpmlint is a tool to check common errors in RPMpackages.
Binary and source packages can be checked.